Web/src/components/join-room/index.tsx

```diff
@@ -95,24 +97,19 @@ const JoinRoomPage: React.FC<JoinRoomProps> = ({ match: { params: { id } } }) =>
     setJoinRequested(true);
 
     setGamePin(newGamePin);
+    setUsername(newUsername);
   };
 
   useEffect(() => {
     if (joinRequested && permission === MotionPermission.NOT_SET) {
       getPermissionAvailability();
     }
     if (joinRequested && permission === MotionPermission.GRANTED) {
-      const name = prompt('What is your name?');
-      if (name == null || name === '') {
-        alert('Name cannot be empty');
-        setJoinRequested(false);
-        return undefined;
-      }
-      comm.joinRoom(gamePin, name);
+      comm.joinRoom(gamePin, username);
       return () => comm.leaveRoom();
     }
     return undefined;
```

Comment:
Is it better to return `undefined` or `() => {};`? I actually did that when I ran into the same situation, although I'm not sure if it's actually better...